digitalutsc/islandora_solution_pack_oralhistories
# Addresses issue 45

# What does this Pull Request do?
Addresses issue #45.  This will mean that there is one less seldom-used feature to test and develop against, thus helping the maintainers and release testers focus on building and maintaining higher priority features.

# What's new?
Removes the side-by-side option from the configuration settings and removes associated code.

# How should this be tested?
Uninstall the module, then reinstall being sure to run the Drupal update script.  After a successful update, you should be able to use the module as usual, but without the option of the side-by-side display.

# Additional Notes:
We should double check that explanations of this feature are removed from the project wiki pages and make sure we document this removal in the release notes for the planned 7.x-1.10 release of this module.

# Interested parties
@Natkeeran @kimpham54
